daprdocs/content/en/developing-applications/local-development/multi-app-dapr-run/multi-app-overview.md

```diff
@@ -0,0 +1,88 @@
+- Recall all of the additional flags you used to tweak the `dapr run` command behavior (`--app-health-check-path`, `--dapr-grpc-port`, `--unix-domain-socket`, etc.)
+
+With Multi-app Run, you can start multiple applications in self-hosted mode using a single `dapr run -f` command using a template file. The template file describes how to start multiple applications as if you had run many separate CLI `run`commands. By default, this template file is called `dapr.yaml`. 
+
+## MapR template file
+
+When you execute `dapr run -f`, uses the multi-app template file to run all the applications. 
+
+You can name template file with preferred name other than the default. For example `dapr run -f ./<your-preferred-file-name>.yaml`.
+
+The following `dapr.yaml` example includes some of the template properties you can customize for your applications. In the example, you can simultaneously launch 2 applications with app IDs of `processor` and `emit-metrics`.
+
+```yaml
+version: 1
+apps:
+  - appID: processor
+    appDirPath: ../../../apps/processor/
+    appPort: 9081
+    daprHTTPPort: 3510
+    command: ["go","run", "app.go"]
+  - appID: emit-metrics
+    appDirPath: ../../../apps/emit-metrics/
+    daprHTTPPort: 3511
+    env: 
+      DAPR_HOST_ADD: localhost
+    command: ["go","run", "app.go"]
+```
+
+For a more in-depth example and explanation of the template properties, see [Multi-app template]({{< ref multi-app-template.md >}}).
+
+## Locations for resources and configuration files
+
+You have options on where to place your applications' resources and configuration files when using MapR.  
+
+### Point to one file location (with convention)
+
+You can set all of your applications resources and configurations at the `~/.dapr` root. This is helpful when all applications share the same resources path, like when testing on a local machine.
+
+### Separate file locations for each application (with convention)
+
+When developing multiple applications, each app directory can have a `.dapr` folder, which contains a `config.yaml` file and a `resources` directory. Otherwise, if the `.dapr` directory is not present within the app directory, the default `~/.dapr/resources/` and `~/.dapr/config.yaml` locations are used.
+
+If you decide to add a `.dapr` directory in each app directory, with a `/resources` directory and `config.yaml` file, you can specify different resources paths for each application. This approach remains within convention by using the default `.dapr`
+
+### Point to separate locations (custom)
+
+You can also name each app directory's `.dapr` directory something other than `.dapr`, such as, `webapp`, or `backend`. This helps if you'd like to be explicit about resource or application directory paths.
+
+## Logs
+
+Logs are included by default within each app directory and are tracked in the following locations under `.dapr/logs`:
+
+- `app.log`
+- `daprd.log`
```

Comment:
- `<appID>_app_<timestamp>.log` (file name format for app's log)
- `<appID>_daprd_<timestamp>.log` (file name format for Darpd log)